content/quickstart/_index.md

```diff
@@ -298,7 +298,7 @@ route = "example.com/*"
 The `route` key here is a [_route pattern_](/about/routes/). Now, we can deploy to the production environment configured above by passing the `--env` flag to `wrangler publish`:
 
 ```console
-wrangler publish --env prod # Publish to example.com
+wrangler publish --env production # Publish to example.com
 wrangler publish            # Publish to workers.dev
```

Comment:
Can you line up the comment indentation between these two lines? That's my only nit here, happy to merge after this :)

Reply:
Sure!